README.md

```diff
@@ -86,6 +86,32 @@ For a main branch:
 $ git diff upstream-code/main main:code
 ```
 
+#### License Check for CHE extensions
```

Comment:
It doesn't look like a separate section, it looks like a **sub**section of the `Updates and branches` section:

<img width="339" alt="image" src="https://github.com/user-attachments/assets/e854b5f7-684a-4194-b5d5-776fc017196b" />

Reply:
@RomanNikitenko Thanks. It was a copy/paste typo. I have fixed it.